test/kad-dht.spec.js

```diff
@@ -73,7 +73,7 @@ function connect (a, b, callback) {
 
 function bootstrap (dhts) {
   dhts.forEach((dht) => {
-    dht.randomWalk._walk(1, 1000, () => {})
+    dht.randomWalk._walk(1, 10000, () => {})
```

Comment:
Now that timeout is actually being passed to `findPeer` I bumped this to 10seconds so the random walk doesn't get killed. This was causing test failures because routing tables weren't doing enough connecting when it was depending on the walk.